rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -93,6 +94,41 @@
 MIN_LOGS_PROCESSED_TO_SLEEP = 1000
 
 
+def decoding_wrapper(
+        msg_aggregator: 'MessagesAggregator | None',
+        transaction_for_logs: EvmTransaction,
+        func: Callable,
+        *args: tuple[Any],
+        **kwargs: Any,
+) -> tuple[Any, bool]:
+    """
+    Wrapper for methods that execute logic from decoders. It handles handles the errors logging
+    them and optionally sending them to the user.
```

Comment:
Note: If you end up removing the if change the docstring here

```suggestion
    Wrapper for methods that execute logic from decoders. It handles all known errors
    by logging them and optionally sending them to the user.
```